### ISSUE TITLE
request -

### ISSUE THREAD

**Issue by sudocode1** [GDColon/GDBrowser#186]

**Comment by GDColon** [comment]
Yeah I'm pretty aware of this, always just used request since that's how I've done it for so long. But if you wanna make a PR feel free to

**Comment by sudocode1** [comment]
yeah i'll work on that at some point - will pr after testing
